rust/error/src/tonic.rs

```diff
@@ -22,6 +22,7 @@ impl From<ErrorCodes> for tonic::Code {
             ErrorCodes::DataLoss => tonic::Code::DataLoss,
             ErrorCodes::Unauthenticated => tonic::Code::Unauthenticated,
             ErrorCodes::VersionMismatch => tonic::Code::Internal,
+            ErrorCodes::UnprocessableEntity => tonic::Code::ResourceExhausted,
```

Comment:
Why does this map?

Reply:
Without it, we're not covering all the possible enumerations of `ErrorCodes` (`UnprocessableEntity` was added)

Reply:
No i mean is unprocessable entity really map to resourceexhasuted? (https://grpc.github.io/grpc/core/md_doc_statuscodes.html)

Reply:
ignore this